internal/cli/args.go

```diff
@@ -148,14 +158,18 @@ func (c *argsCmd) run(ctx context.Context, args []string) error {
 	a.Incr("cmd.args", tags.AsMap())
 	defer a.Flush(time.Second)
 
+	if sliceutils.StringSliceEquals(tf.Spec.Args, args) {
+		logger.Get(ctx).Infof("Tilt is already running with those args. No action taken.")
```

Comment:
It seems concerning to have capitalization and punctuation.   Below log message don't have it... this has it.  It is unclear when it is appropriate or not but seems at this point inconsistent.